py/torch_tensorrt/dynamo/backend/__init__.py

```diff
@@ -8,10 +8,10 @@
 from torch_tensorrt import EngineCapability, Device
 from torch_tensorrt.fx.utils import LowerPrecision
 
-from torch_tensorrt.dynamo.backend._settings import CompilationSettings
+from torch_tensorrt.dynamo.common import CompilationSettings
 from torch_tensorrt.dynamo.backend.utils import prepare_inputs, prepare_device
 from torch_tensorrt.dynamo.backend.backends import torch_tensorrt_backend
-from torch_tensorrt.dynamo.backend._defaults import (
+from torch_tensorrt.dynamo.common._defaults import (
```

Comment:
same here

Reply:
This one, I am more open to moving up a level since the necessary changes have been made to _both_ compile and export. I could add `from torch_tensorrt.dynamo.common._defaults import _defaults` to the `torch_tensorrt.dynamo.__init__.py`